pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -448,13 +462,15 @@ func (a *Operator) ensureTenantRBAC(operatorNamespace, targetNamespace string, c
 
 		// role doesn't exist, create it
 		// TODO: we can work around error cases here; if there's an un-owned role with a matching name we should generate instead
-		ownedRole.SetNamespace(targetNamespace)
-		ownedRole.SetOwnerReferences([]metav1.OwnerReference{ownerutil.NonBlockingOwner(targetCSV)})
-		if err := ownerutil.AddOwnerLabels(ownedRole, targetCSV); err != nil {
+		targetRole := ownedRole.DeepCopy()
+		targetRole.SetResourceVersion("0")
```

Comment:
the resourceversion being set was causing creates to fail

honestly not sure why this was a flake and not just a flat out failure. I meant to go back and see why this ever worked.

Reply:
This makes me think we need some kind of "SafeDeepCopy".

Reply:
That's a good idea - like a `CopyForCopy` vs. `CopyForUpdate`